shell/browser/extensions/api/streams_private/streams_private_api.cc

```diff
@@ -24,7 +24,8 @@ void StreamsPrivateAPI::SendExecuteMimeTypeHandlerEvent(
     bool embedded,
     int frame_tree_node_id,
     blink::mojom::TransferrableURLLoaderPtr transferrable_loader,
-    const GURL& original_url) {
+    const GURL& original_url,
+    const std::string& internal_id) {
```

Comment:
It looks like this implementation may have diverged from upstream, we should check it in a followup!